### PR TITLE
Merge back: `develop` => `main`

### PR DESCRIPTION
**Pull request by brianfeister**
Prior to landing the last (currently) open PR for the Seshu Handoff https://github.com/meetnearme/api/pull/60, we want to promote `develop` to `main`. The seshu handoff PR is intended to be deployed / promoted to `develop` as part of testing and continued work, but we don't want all of those open `TODO:`s in `main`.

Hence, we want to merge back now.

### CODE REVIEW

File: functions/gateway/handlers/page_handlers.go [hunk 1]
```diff
@@ -13,20 +14,19 @@ import (
...
-var mw   *authentication.Interceptor[*openid.UserInfoContext[*oidc.IDTokenClaims, *oidc.UserInfo]]
+var mw *authentication.Interceptor[*openid.UserInfoContext[*oidc.IDTokenClaims, *oidc.UserInfo]]
 
-func init () {
-	Db = transport.GetDB()
-	mw, _ = services.GetAuthMw()
+var bypassAuthForTesting bool
+
+func SetBypassAuthForTesting(bypass bool) {
+    bypassAuthForTesting = bypass
```
Comment (Brandon-G-Tripp) [inline functions/gateway/handlers/page_handlers.go:29]: I will double check but I believe we can remove as was never fully implemented.

Comment (brianfeister) [reply functions/gateway/handlers/page_handlers.go:29]: Alright, I'll look for a new PR to `develop` to fix this before we promote to `main`

**Review by Brandon-G-Tripp — COMMENTED** [review]
This looks good. I am just going to remove the small unused code that I had flagged.

**Comment by brianfeister** [comment]
> This looks good. I am just going to remove the small unused code that I had flagged.

@Brandon-G-Tripp open a PR aimed at `develop` if you're fixing that. When it merges to `develop` it automatically will appear in the commit change set in this PR.

You don't want to commit it directly to `develop` (which is how it would wind up in this PR)

**Review by Brandon-G-Tripp — APPROVED** [review]
Ok the clean up is finished. This is good to promote to main.

**Pull request merged**